Portfolio.html

```diff
@@ -352,7 +247,11 @@ <h1 class="contact-title">Contact me</h1>
         that you need built or a project that needs coding. I’d love to help
         with it
       </p>
-      <form action="https://formspree.io/f/" class="contact-form" method="POST">
+      <form
+        action="https://formspree.io/f/mvollnwk"
+        class="contact-form"
+        method="POST"
+      >
         <ul class="inputs">
```

Comment:
- [ ]  As the previous reviewer suggested, try to fix your popup so that it can fit on both desktop and mobile devices referring to Figma design. 
![image](https://user-images.githubusercontent.com/43877689/175652330-70027481-447d-468a-8a5e-808ea9e600ce.png).